package.json

```diff
@@ -24,28 +24,27 @@
   },
   "devDependencies": {
     "@babel/plugin-transform-react-jsx": "^7.10.4",
-    "@fewlines/eslint-config": "2.1.1",
-    "@fewlines/fwl-logging": "^1.0.0",
+    "@fewlines/eslint-config": "^3.0.0",
     "@testing-library/jest-dom": "^5.11.1",
     "@testing-library/react": "^10.4.7",
     "@types/jest": "^26.0.4",
     "@types/node": "14.0.23",
     "@types/react": "16.9.43",
     "@types/react-dom": "16.9.8",
     "@types/styled-components": "5.1.1",
-    "@typescript-eslint/eslint-plugin": "3.6.1",
-    "@typescript-eslint/parser": "3.6.1",
+    "@typescript-eslint/eslint-plugin": "^3.8.0",
+    "@typescript-eslint/parser": "^3.8.0",
     "babel-jest": "^26.1.0",
     "babel-plugin-styled-components": "^1.10.7",
-    "eslint": "7.4.0",
-    "eslint-config-prettier": "6.11.0",
-    "eslint-plugin-import": "2.22.0",
-    "eslint-plugin-prettier": "3.1.4",
-    "eslint-plugin-react": "7.20.3",
+    "eslint": "^7.6.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-import": "^2.22.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react": "^7.20.5",
     "file-loader": "6.0.0",
     "jest": "^26.1.0",
     "jest-fetch-mock": "^3.0.3",
-    "prettier": "2.0.5",
+    "prettier": "^2.0.5",
     "ts-jest": "^26.1.2",
     "ts-node": "8.10.2",
     "typescript": "3.9.6",
```

Comment:
I have not told you yet, but we want full control over dependencies versions. For that, we remove the `^` from the version. Could you also remove them? Thanks! 😁

Reply:
OK I  have rolled back the package.json file to its previous version 18e6418

Reply:
Thanks mate! I'll handle the dependencies bump (I have a recurring task for that, set to once a month I think?). Just keep in mind to remove the `^` when you add new dependencies :).